patches/PSP.cmake

```diff
@@ -1,34 +1,66 @@
-SET(CMAKE_SYSTEM_NAME Generic)
-SET(CMAKE_SYSTEM_VERSION 1)
-SET(CMAKE_CROSSCOMPILING TRUE)
+#
+# CMake toolchain file for PSP.
+#
+# Copyright 2019 - Wally
+# Copyright 2020 - Daniel 'dbeef' Zalega
 
-# set compiler
-set(CMAKE_C_COMPILER psp-gcc)
-set(CMAKE_CXX_COMPILER psp-g++)
+# Determine PSP toolchain installation directory (psp-config binary is added to path after successful installation):
```

Comment:
Might want to check [$ENV{PSPDEV}](https://cmake.org/cmake/help/latest/variable/ENV.html) before. Should be defined for most and saves one executable call.

Reply:
I want this process to be simple and deterministic, no branching like - if not defined in environment, then call psp-config, then if no psp-config result, check if variable was passed to cmake call... 

So I would stick with psp-config, as from what I see it's de-facto standard of checking psp toolchain location in many projects.